### PR TITLE
Update Python "verbose" param in WebDriver article

### PR DESCRIPTION
**Pull request by Delta456**
Rendered article section for review:
* **Use WebDriver to automate Microsoft Edge** > **Manage and configure the Microsoft Edge WebDriver service**
   * https://review.learn.microsoft.com/microsoft-edge/webdriver-chromium/?branch=pr-en-us-3303&tabs=python#manage-and-configure-the-microsoft-edge-webdriver-service
   * [Before/Live](https://learn.microsoft.com/microsoft-edge/webdriver-chromium/&tabs=python#manage-and-configure-the-microsoft-edge-webdriver-service)
   * changelist

The parameter `verbose` in `Service` has been [removed](https://github.com/SeleniumHQ/selenium/blob/trunk/py/selenium/webdriver/edge/service.py) and the code no longer works. This PR updates the coding to a working stage.

AB# pending

### CODE REVIEW

**Review by leotlee — APPROVED** [review]
LGTM - Dev

**Review by mikehoffms — APPROVED** [review]
LGTM - Writer (approving after Leo, to add a green checkmark to enable Merge button)

**Pull request merged**
